app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/Evaluate.tsx

```diff
@@ -1,13 +1,25 @@
 import { MuiButton } from "@fiftyone/components";
 import { Add } from "@mui/icons-material";
+import { Box } from "@mui/material";
 import React from "react";
 
 export default function Evaluate(props: EvaluateProps) {
-  const { onEvaluate } = props;
+  const { onEvaluate, permissions } = props;
+  const canEvaluate = permissions.can_evaluate;
```

Comment:
How where is this permission set?

Reply:
Permissions are initialized on loading the panel in file `fiftyone/operators/builtins/panels/model_evaluation/__init__.py`